util/unifyfs/src/unifyfs-rm.c

```diff
@@ -1019,22 +1019,24 @@ static int srun_launch(unifyfs_resource_t* resource,
     size_t argc, srun_argc, server_argc;
     char** argv = NULL;
     char n_nodes[16];
+    char n_cores[8];
 
-    // full command: srun <srun args> <server args>
-
-    srun_argc = 5;
-    snprintf(n_nodes, sizeof(n_nodes), "%zu", resource->n_nodes);
+    snprintf(n_cores, sizeof(n_cores), "-c%d", resource->n_cores_per_server);
+    snprintf(n_nodes, sizeof(n_nodes), "-N%zu", resource->n_nodes);
 
+    // full command: srun <srun args> <server args>
+    srun_argc = 6;
     server_argc = construct_server_argv(args, NULL);
 
     // setup full command argv
     argc = 1 + srun_argc + server_argc;
     argv = calloc(argc, sizeof(char*));
     argv[0] = strdup("srun");
-    argv[1] = strdup("-N");
-    argv[2] = strdup(n_nodes);
-    argv[3] = strdup("--ntasks-per-node");
-    argv[4] = strdup("1");
+    argv[1] = strdup("--exact");
+    argv[2] = strdup("--overcommit");
```

Comment:
Thanks for fixing this. This is now required to run on our end as well, as the SLURM default apparently changed last week.

Reply:
It's good to know it works on other current SLURM systems. I believe it should work on systems running older versions as well.